Added steps for anyone who is interested our project's  collaborate task.

Amutha37 opened the pull request:
## Overview of the Pull Request:
_Describe your changes here..._

List a few steps on how to joint our project for collaboration.
We have pre existing steps for working in the forked repo not for how to forked it and start the collaboration  which should be initial steps.


## link to Trello card or Github issue

## How Has This Been Tested?
- Please describe the tests that you ran to verify your changes.
. these  are just README.md notes showing steps for  anyone new and interested in this collaboration work to follow the  steps  listed.

- Provide instructions so we can reproduce. 
. Please read the steps and confirm they are all  in correct orders.

- If necessary, please also list any relevant details for your test setup/configuration
- no setup test for README notes.

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ ✔️] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [ ✔️ ] Is your pull request up-to-date with `main` branch?
- [ ✔️ ] Have you written instructions on how to test that your changes work as expected?
- [ ✔️ ] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

amu3 committed:
add not if anyone want to collaborate in this project and steps

OlaShabalina committed:
Merge branch 'staging' into ReadmeUpdate

OlaShabalina commented:
Hi @Amutha37 , great thought on updating the instructions. I think your steps are clear. However, should we just keep steps 1, 2 then 6 (as 3rd step)? For majority of people you never need to set up upstream as it will be set to default branch anyway (in our case `staging`). The only time you play with upstream is when you base your changes on a different branch. I assume it might have happened to you as you used master/main branch without realising it's now `staging`.

OlaShabalina commented on README.md:
```diff
@@ -83,6 +83,38 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
+
+2. Open terminal `git clone ...` your forked repository .
+
+3. List the current configured remote repository for your fork.
+   `git remote -v`
+
+   > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
+   > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
+
+4. Specify a new remote upstream repository that will be synced with the fork.
+   ` git remote add upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git`
+
+5. Verify the new upstream repository you've specified for your fork.
+   `git remote -v`
+
+   > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
+   > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
+   > upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (fetch)
+   > upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (push)
+
+6. `git checkout -b (name of the branch)` Now you can work on the branch.
```
```suggestion
3. `git checkout -b (name of the branch)` Now you can work on the branch.
```

Hi @Amutha37 , great thought on updating the instructions. I think your steps are clear. However, should we just keep steps 1, 2 then 6 (as 3rd step)? For majority of people you never need to set up upstream as it will be set to default branch anyway (in our case `staging`). The only time you play with upstream is when you base your new branch on a different branch. I assume it might have happened to you as you used `main` branch without realising the default is now `staging`.

Amutha37 replied:
Thank you for the feedback. Yes, I agree with you . The question is what if we at sill at 'main' branch and not in 'staging' . Like we was working initially . Is this something I should consider in the README before removing those lines that you feel not necessary.

hqtan replied:
Hi @Amutha37 ,

I think this conversation has gone on unresolved for too long. I'll resolve this for now, and address outstanding work in a separate PR.

OlaShabalina commented on README.md:
```diff
@@ -83,6 +83,38 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 
 </details>
 
+## Collaboration: Want to collaborate with this project ?
```
```suggestion
## Collaboration: Want to collaborate on this project ?
```

OlaShabalina commented on README.md:
```diff
@@ -83,6 +83,38 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
+STEPS
+</summary>
+
+1. Forked the project repository from the owners repository.
```
```suggestion
1. Fork the project repository from this repository.
```

Amutha37 replied:
I have corrected  the suggested changes. Thank you

OlaShabalina replied:
It still shows the original text @Amutha37 , do you want to push the changes?

hqtan replied:
Hi @Amutha37 ,

I think this conversation has gone on unresolved for too long. I'll resolve this for now, and address outstanding work in a separate PR.

hqtan commented:
Hi @Amutha37,

Thanks again for updating the instructions in the Readme. I generally agree with @OlaShabalina's suggestions...

But, I wonder if we could save ourselves the trouble of explaining how to fork a repo (and sharing changes from the fork back upstream), and point readers to [this link](https://docs.github.com/en/get-started/quickstart/fork-a-repo) instead? That link does a **much** better job of explaining; and it has pretty pics too!

OlaShabalina commented:
Actually, @Amutha37 , there is another sections called `collaboration`, is your change replacing that section?

hqtan committed:
Merge branch 'staging' into ReadmeUpdate

Amutha37 committed:
Update README.md

Acceted changes

Co-authored-by: Olga <88268603+OlaShabalina@users.noreply.github.com>

Amutha37 commented:
The steps working in forked repo is after the initial steps . The Initial steps are missing that is what I was adding.

hqtan committed:
Merge branch 'staging' into ReadmeUpdate

Amutha37 commented:
The link to the doc Qing directing is enough.
I can add that in the README.

OlaShabalina committed:
Merge branch 'staging' into ReadmeUpdate

amu3 committed:
set a github doc link for collobartion steps.

hqtan commented:
Hi @Amutha37 ,

You have not [adequately addressed the concerns that @OlaShabalina raised yet](https://github.com/CodeforAustralia/council-emissions-calculator/pull/241#issuecomment-1432997512).
You have **added** a new section called 'Collaboration: Want to collaborate on this project ?', rather than **update** an existing section called 'Collaboration: Working in a Forked Repository'. Please review your changes.

Amutha37 commented:
I edited my notes for Olga. The steps are different to each other.

Amutha37 commented:
I have added new steps and updated existing steps. 

I have replaced the link you suggested for initial steps I adde d made small changes to the existing steps. 

I have have edited my reply to Olga's question too. Hope this is much clearer.
